waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -1117,7 +1127,7 @@
    configuration against kubernetes-scheduler-schema and throws if it's not valid."
   [{:keys [authentication authorizer cluster-name container-running-grace-secs custom-options http-options log-bucket-sync-secs
            log-bucket-url max-patch-retries max-name-length pod-base-port pod-sigkill-delay-secs
-           pod-suffix-length replicaset-api-version replicaset-spec-builder restart-expiry-threshold scheduler-name
+           pod-suffix-length proxy-options replicaset-api-version replicaset-spec-builder restart-expiry-threshold scheduler-name
```

Comment:
we should add necessary validation on `proxy-options` in the `pre` precondition checks of the function.